ImportSigningKey throws segfault when importing Key with PCR values

I'm trying to use `ImportSigningKey` after sealing an RSA key using `CreateSigningKeyImportBlob` on my laptop.  I specified PCR values i would expect on the TPM system that currenly has:

```
$ tpm2_pcrread sha256:23
sha256:
  23: 0xF5A5FD42D16A20302798EF6ED309979B43003D2320D9F0E8EA9831A92759FB4B
```


THe following program is run on my laptop where i send in arguments to encode the string into bytes and finally as the expected(?) format of the `CreateSigningKeyImportBlob` method

I used a GCP serviceaccount .p12 file here but you can ofcourse use even a transient RSA key for this `rsa.GenerateKey(rand.Reader, 2048)`

TO repro, se any PCR value and create the keyblob using

-"laptop"
  https://gist.github.com/salrashid123/a252d76235d334de2ea7b849695a6f58

```
go run main.go  \
  --rsaKeyFile=svc_account.p12 \
  --sealedOutput=sealed.dat \
  --pcrValues=23=f5a5fd42d16a20302798ef6ed309979b43003d2320d9f0e8ea9831a92759fb4b -v=2 -alsologtostderr
```

---

The output is a `sealded.dat` which you transfer to the VM with the tpm.

On that, run the following program to import the blob 

https://gist.github.com/salrashid123/71c390c41578ab4ea4da3131f312ca60

as 

```
go run main.go  \
  --importSigningKeyFile=sealed.dat \
  --keyHandleOutputFile=key.dat \
  --v=2 -alsologtostderr
```

you'll see it prints the existing on-system PCR corrently but still throws the segfault at

```
key, err := ek.ImportSigningKey(importblob)
```

---
```
./import  --importSigningKeyFile=sealed.dat --keyHandleOutputFile=key.dat --v=2 -alsologtostderr
I0302 23:07:41.203882    9040 main.go:49] ======= Init importSigningKey ========
I0302 23:07:41.239047    9040 main.go:71] Handle 0x80000001 flushed
I0302 23:07:41.239073    9040 main.go:76] ======= Print PCR 23 ========
I0302 23:07:41.239783    9040 main.go:81] PCR23: 00000000  f5 a5 fd 42 d1 6a 20 30  27 98 ef 6e d3 09 97 9b  |...B.j 0'..n....|
00000010  43 00 3d 23 20 d9 f0 e8  ea 98 31 a9 27 59 fb 4b  |C.=# .....1.'Y.K|
I0302 23:07:41.239805    9040 main.go:83] ======= Loading EndorsementKeyRSA ========
I0302 23:07:41.245614    9040 main.go:90] ======= Loading sealedkey ========
I0302 23:07:41.245938    9040 main.go:101] ======= Loading ImportSigningKey ========
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x80 pc=0x5fab09]

goroutine 1 [running]:
github.com/google/go-tpm-tools/tpm2tools.(*Key).Close(0x0)
	/apps/go/go_path/pkg/mod/github.com/google/go-tpm-tools@v0.1.3-0.20200229023135-1b9abc4c9ab8/tpm2tools/keys.go:197 +0x29
github.com/google/go-tpm-tools/tpm2tools.(*Key).ImportSigningKey.func1(0xc00009db48, 0xc00009db40)
	/apps/go/go_path/pkg/mod/github.com/google/go-tpm-tools@v0.1.3-0.20200229023135-1b9abc4c9ab8/tpm2tools/import.go:72 +0x43
github.com/google/go-tpm-tools/tpm2tools.(*Key).ImportSigningKey(0xc0000f2000, 0xc0000940e0, 0x0, 0x6c8740, 0xc00001ad14)
	/apps/go/go_path/pkg/mod/github.com/google/go-tpm-tools@v0.1.3-0.20200229023135-1b9abc4c9ab8/tpm2tools/import.go:80 +0x2d3
main.importSigningKey(0x67e1f9, 0x9, 0x7ffea8ffb842, 0xa, 0x7ffea8ffb863, 0x7, 0x0, 0x0)
	/home/srashid/Desktop/tpm_poc/go_import_rsa/transfer/shieldedvm/main.go:102 +0xb0e
main.main()
	/home/srashid/Desktop/tpm_poc/go_import_rsa/transfer/shieldedvm/main.go:41 +0xd6
```

--- a/tpm2tools/import.go
+++ b/tpm2tools/import.go
@@ -73,10 +73,10 @@ func (k *Key) ImportSigningKey(blob *tpmpb.ImportBlob) (key *Key, err error) {
 	}()
 
 	if key.pubArea, _, _, err = tpm2.ReadPublic(k.rw, handle); err != nil {
-		return nil, err
+		return
 	}
 	if key.session, err = newPCRSession(k.rw, PCRSelection(blob.Pcrs)); err != nil {
-		return nil, err
+		return
 	}
 	return key, key.finish()
 }

